### ISSUE TITLE
IoP Wallet Android- No tiene limite de digitos en Requestes amount 

### ISSUE THREAD

**Issue by andreagmarcano** [Fermat-ORG/beta-testing-program#854]
#### Nombre pantalla

IoP Wallet

### Nombre de la Aplicación 

IoP version 1.0.2

-----

##### Reporte de error: 

Los caracteres deberían ser limitados ya que deja escribir cualquier cantidad de números sin limite alguno. 

Como recomendación, también seria de utilidad que los digitos tengan todos el mismo tamaño.

##### Screenshot o Capture de pantalla:

 
![iop](https://cloud.githubusercontent.com/assets/18452495/19156260/3f82953a-8baf-11e6-88a6-2843853eb599.PNG)

#### Dispositivo desde donde se realizo la prueba

![movil](https://cloud.githubusercontent.com/assets/18452495/19156273/4e2198c0-8baf-11e6-8b7e-2ae92e4cb45d.PNG)
![dpi](https://cloud.githubusercontent.com/assets/18452495/19156279/51bd7af8-8baf-11e6-9dee-69ac871b7dba.PNG)

**Comment by MaviDavsa** [comment]
@furszy

**Comment by furszy** [comment]
No, el usuario puede gastar lo que quiera gastar y con la exactitud que necesite. No es lo mismo que algo cueste 1,564 dolares que algo cueste 1,5.

**Comment by AdryCanonico** [comment]
Te entiendo pero creo que un limite de caracteres en milésimas seria bueno
